app/src/main/java/org/mozilla/fenix/perf/StartupTimeline.kt

```diff
@@ -77,7 +77,6 @@ object StartupTimeline {
  */
 @OptIn(DelicateCoroutinesApi::class) // GlobalScope usage
 internal class StartupHomeActivityLifecycleObserver(
-    private val frameworkStartMeasurement: StartupFrameworkStartMeasurement,
     private val startupTimeline: PingType<NoReasonCodes> = Pings.startupTimeline,
```

Comment:
thought (non-blocking): this code is scheduling and submitting a telemetry ping: `Pings.startupTimeline`. A telemetry ping is a group of telemetry metrics/probes bundled and sent to the server together. With your changes, you actually removed all of the metrics in the startup-timeline ping – https://searchfox.org/mozilla-mobile/search?path=%5Efenix&q=startup-timeline – so we can remove the 1) the startup-timeline ping from `pings.yaml` and 2) remove the code that schedules/submits it (i.e. this code right here).

You could either add that change to this PR, add it to a follow-up PR (to keep it simple), or, if you'd rather work on something else, file a follow-up bug to remove it that someone else can work on. Your choice! 😄

Reply:
Let me know if I got this correct, since I removed `startup-timeline metrics` , so now there is no metrics to send. Now since there no metrics, there is no need to Ping the server for metrics ?
Am I right here ?

Reply:
Update this has been removed

Reply:
> since I removed startup-timeline metrics , so now there is no metrics to send. Now since there no metrics, there is no need to Ping the server for metrics ?
Am I right here ?

Yep! That's correct!